src/pages/signin/SAMLSignInPage/index.native.tsx

```diff
@@ -42,7 +62,12 @@ function SAMLSignInPage() {
             if (searchParams.has('error')) {
                 Session.clearSignInData();
                 Session.setAccountError(searchParams.get('error') ?? '');
-                Navigation.navigate(ROUTES.HOME);
+
+                Navigation.isNavigationReady().then(() => {
+                    // We must call goBack() to remove the /transition route from history
```

Comment:
Just curious - where is this transition route set? Is it only from OldDot -> NewDot?